numpy/distutils/clang_cl.py

```diff
@@ -0,0 +1,167 @@
+"""
+Build/Install NumPy on Windows with clang-cl using (powershell syntax)
+
+Short:
+python setup.py build --compiler=clang-cl install
+
+Full:
+python setup.py config --compiler=clang-cl `
+                build_clib --compiler=clang-cl `
+                build_ext --compiler=clang-cl `
+                install
+
+Develop mode:
+python setup.py build_ext -i --compiler=clang-cl develop
```

Comment:
Missing a `config` here? `setuptools` shouldn't understand a `--compiler` flag to `setup.py`.

Reply:
Thanks. Fixed now and tested.